kod/object/passive/skill/disarm.kod

```diff
@@ -140,7 +140,17 @@ messages:
          {
             % Currently, have 1/5th your percent plus your modifier chance to make opponent drop weapon.
             iAbil = (iAbil + 1 + iModifier) / DISARM_FACTOR;
-            if (random(1,100) < iAbil) and (not Send(Send(oTarget,@GetOwner),@IsArena))
+
+            % Cannot force a weapon to drop if...
```

Comment:
This comment is too explicit; it simply repeats the code and will thus become out of date if the code ever changes.  What the reader wants to know is *why* are these checks here.